Source/WebCore/css/ShorthandSerializer.cpp

```diff
@@ -182,6 +185,20 @@ bool ShorthandSerializer::subsequentLonghandsHaveInitialValues(unsigned startInd
     return true;
 }
 
+bool ShorthandSerializer::commonSerializationChecks(const ComputedStyleExtractor& properties)
+{
+    ASSERT(length());
+
+    if (m_shorthand.id() == CSSPropertyAll)
+        return false;
```

Comment:
Why is this needed? What happens if you remove it? Can it be an assertion instead?